05_syntax_lab/02.py

```diff
@@ -1,6 +1 @@
-"""
-Write a Python program that randomizes 7 numbers
-and prints their sum total.
-If the sum is divisable by 7, also print the word "Boom"
-"""
-
+my solution
```

Comment:
comment after pressing + on the last line of the 02.py examples.
Please keep the description of the exercises at the top of the file. It helps us remember what was the exercises about, and it will help you too. Oh and it is also a good practice to have a description of your program at the top of file.